SDWebImage/SDAnimatedImageView.m

```diff
@@ -239,20 +233,11 @@ - (void)setImage:(UIImage *)image
         
         [self.layer setNeedsDisplay];
 #if SD_MAC
-        [self.layer displayIfNeeded]; // macOS's imageViewLayer is not equal to self.layer. But `[super setImage:]` will impliedly mark it needsDisplay. We call `[self.layer displayIfNeeded]` to immediately refresh the imageViewLayer to avoid flashing
+        [self.layer displayIfNeeded]; // macOS's imageViewLayer may not equal to self.layer. But `[super setImage:]` will impliedly mark it needsDisplay. We call `[self.layer displayIfNeeded]` to immediately refresh the imageViewLayer to avoid flashing
 #endif
     }
 }
 
-- (void)setAnimationRepeatCount:(NSInteger)animationRepeatCount
-{
```

Comment:
This, actually does nothing now. Just remove it. If we have custom override logic, then we can add it back.